ofl/notosansnko/METADATA.pb

```diff
@@ -1,19 +1,25 @@
-name: "Noto Sans N Ko"
+name: "Noto Sans NKo"
```

Comment:
It should match the previous font. This would count as a CamelCase name, which is not allowed on a regular basis. Less of all here, given the previous name was `N Ko`